### DISCUSSION TITLE
讓python當接歌DJ

### DISCUSSION THREAD

**Discussion by wu51000** [1101-GS4719-C/Python-Community-GS4719-C#1107] [優秀Python套件]
我是一個非常喜歡音樂的人
可以說是無樂不歡
歌單裡非常多歌
但每次前後曲風差異太大讓人聽了沒有那麼爽快
於是我發現有能按bpm（音樂速度）排序的python套件
十分期待
[https://zhuanlan.zhihu.com/p/265124519](url)
https://zhuanlan.zhihu.com/p/265124519

**Comment by BlahFish1006** [comment]
好酷！感謝分享，感覺有空可以自己來試試玩玩看~
看介紹似乎code並沒有很複雜，但感覺還挺實用，找到除了透過付費的音樂串流平台和youtube播放清單的第三條路，
可以透過所學將日常所用進行更加個人化的處理，感覺會很有趣~

**Reply by wu51000** [reply]
（題外話
我真的覺得YouTube好賊喔
免費這麼多年突然來個YouTube premium
太高招了 被打個措手不及……

**Comment by wenchi9023** [comment]
我在不同時候想聽的曲風類型不一樣，有時候只想聽抒情歌，運動的時候想聽節拍較重、較輕快的歌
有了這個套件真的很方便欸，這樣就不用一直切歌了~

**Reply by wu51000** [reply]
希望我的分享有幫助到你～～
小小好奇你的歌單有哪些哇哈哈

**Comment by dinosaur1009** [comment]
這個技術感覺音樂軟體可以參考！更能貼近客群

**Reply by wu51000** [reply]
嗅到商機……
不要像YouTube一樣賊我什麼都好qq

**Comment by Sylvie0601** [comment]
我覺得超級棒的ㄟ ! 如果有這個被發展成App的話我就可以把spotify丟掉了哈哈
而且這樣就可以不用在歌單裡面切來切去的 !
